.github/workflows/ci.yaml

```diff
@@ -19,9 +19,18 @@ jobs:
       # TODO remove once testcontainers are updated
       - name: pull ryuk image
         run: docker pull testcontainersofficial/ryuk:0.3.0
+      - name: Build docker image
+        uses: docker/build-push-action@v2.7.0
+        # We need the image for the tests
+        with:
+          context: .
+          push: false
+          tags: ghcr.io/jaegertracing/spark-dependencies/spark-dependencies
```

Comment:
why spark-dependencies/spark-dependencies twice?

Reply:
This builds the docker image so it can be tests in tests.
I do not think executing "docker exec" is the right thing to do from `*Test.java` code as it would be complicated

Note: this does not push the image, and it just builds it locally.

Just for your reference, Testcontainers can build image on the fly, however, it does not use `docker` cli, so it can't cope with `--mount=...` lines: https://github.com/testcontainers/testcontainers-java/issues/2857

Reply:
My question was why the odd double name?